src/main/java/redis/clients/jedis/resps/ClusterShardNodeInfo.java

```diff
@@ -0,0 +1,91 @@
+package redis.clients.jedis.resps;
+
+import java.util.Map;
+
+/**
+ * This class holds information about a node of the cluster with command {@code CLUSTER SHARDS}.
+ * They can be accessed via getters. There is also {@link ClusterShardNodeInfo#getClusterShardNodeInfo()}
+ * method that returns a generic {@link Map} in case more info are returned from the server.
+ */
+public class ClusterShardNodeInfo {
+
+	public static final String ID = "id";
+	public static final String ENDPOINT = "endpoint";
+	public static final String IP = "ip";
+	public static final String HOSTNAME = "hostname";
+	public static final String PORT = "port";
+	public static final String TLS_PORT = "tls-port";
+	public static final String ROLE = "role";
+	public static final String REPLICATION_OFFSET = "replication-offset";
+	public static final String HEALTH = "health";
+
+	private final String id;
+	private final String endpoint;
+	private final String ip;
+	private final String hostname;
+	private final Long port;
+	private final Long tlsPort;
+	private final String role;
+	private final Long replicationOffset;
+	private final String health;
+
+	private final Map<String, Object> clusterShardNodeInfo;
+
+	/**
+	 * @param map contains key-value pairs with node info
+	 */
+	public ClusterShardNodeInfo(Map<String, Object> map) {
```

Comment:
Please format all tab characters in this class with space characters.

Reply:
Done